versionist.conf.js

```diff
@@ -38,7 +38,11 @@ module.exports = {
   },
 
   getIncrementLevelFromCommit: (commit) => {
-    return commit.footer['Change-Type']
+    if (/none/i.test(commit.footer['Change-Type'])) {
```

Comment:
If Etcher now "officially supports" `Change-Type: none` (it didn't before), should we also update https://github.com/resin-io/etcher/blob/master/docs/COMMIT-GUIDELINES.md#change-type-type ?

Reply:
I merged this so we can try to release today, but I agree a follow-up PR with such information would be nice.